src/pages/MergeReadiness/index.tsx

```diff
@@ -0,0 +1,349 @@
+          <FormattedMessage defaultMessage="The long awaited transition to proof-of-stake via the Merge is rapidly approaching, bringing Ethereum one step closer to a more sustainable ecosystem." />
+        </Text>
+        <Text className="mt20">
+          <FormattedMessage defaultMessage="Before the switch, there are a couple things solo stakers and validator services need to be aware of." />
+        </Text>
+      </Subtitle>
+      <ChecklistPageStyles>
+        <section>
+          <Heading level={3} id="el-plus-cl">
+            <FormattedMessage defaultMessage="EL + CL = Ethereum" />
+          </Heading>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="Since the genesis of the Beacon Chain, many validators running their own consensus layer (CL) client have opted to use third-party services for their execution layer (EL) connection. This has been acceptable since the only thing being listened to was the staking deposit contract." />
+          </Text>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="With the Merge, the burden of processing transactions will fall on validators, as proof-of-work is deprecated. To sign off on the validity of these transactions, a validator must have access to the events of the execution layer." />
+          </Text>
+          <Alert variant="error" className="my40">
+            <Text>
+              <FormattedMessage defaultMessage="At time of the Merge, Infura and Alchemy will no longer be viable options to outsource execution layer responsibilities." />
+            </Text>
+            <Text className="mt20">
+              <FormattedMessage defaultMessage="To prevent downtime at time of the Merge, be sure your node is running both an EL client, as well as a CL client." />
+            </Text>
+          </Alert>
+          <Text className="mb10">
+            <FormattedMessage defaultMessage="Stakers running their own execution layer is necessary for the decentralization of the network." />
+          </Text>
+          <CheckBox
+            label={
+              <Text className="checkbox-label">
+                <FormattedMessage defaultMessage="I am running my own execution layer client." />
+              </Text>
+            }
+          />
+          <CheckBox
+            label={
+              <Text className="checkbox-label">
+                <FormattedMessage defaultMessage="I am running my own consensus layer client." />
+              </Text>
+            }
+          />
+        </section>
+        <section>
+          <Heading level={3} id="fee-recipient">
+            <FormattedMessage defaultMessage="Fee recipient" />
+          </Heading>
+          <Text className="mt20">
+            <FormattedMessage defaultMessage="Now that transactions must be processed by validators, the validators that propose blocks including these transactions are eligible to receive the unburnt transaction fees." />
```

Comment:
just curious, do most people call it "unburnt transaction fees" or "tips"?

Reply:
Good question, curious peoples take on this as well. I've found "unburnt" can make this more straight forward for people who think all the fees are burnt. 

Again, could go either way, just want it to be clear to people.

Reply:
Just pushed a copy adjustment here to be more inclusive, depending on what the user may be used to.